internal/ingest/ingest.go

```diff
@@ -154,50 +165,32 @@ func (ing *Ingester) Close() error {
 // synced in the background.  The completion of advertisement sync does not
 // necessarily mean that the entries corresponding to the advertisement are
 // synced.
-func (ing *Ingester) Sync(ctx context.Context, peerID peer.ID, peerAddr multiaddr.Multiaddr, depth int64) (<-chan multihash.Multihash, error) {
-	log := log.With("peerID", peerID)
+func (ing *Ingester) Sync(ctx context.Context, peerID peer.ID, peerAddr multiaddr.Multiaddr, depth int64, nolatest bool) (<-chan multihash.Multihash, error) {
```

Comment:
I had only meant to use `nolatest` for the CLI flag and the API query parameter string.  Here, standard golang conventions apply, so `ignoreLatest` is OK.

Anyway, just a minor clarification, code is good either way you prefer.

Reply:
Sure thanks Andrew